CHANGELOG.rst

```diff
@@ -1,6 +1,77 @@
 Changelog
 =========
 
+0.9.0 (2019-03-13)
+------------------
+
+Major features:
+
+- Support for running MLflow Projects in Docker containers. This allows users to include non-Python dependencies in their project environments and provides stronger isolation when running projects. See the `Projects documentation <https://mlflow.org/docs/latest/projects.html>`_ for more information. (#555, @marcusrehm; #819, @mparkhe; #970, @dbczumar)
+- SqlAlchemyStore: new backend store implementation using SQLAlchemy for MLflow Tracking Server. Support for a scalable and performant backend store was one of the top community requests. This community contribution feature enables users to connect to local or remote SQLAlchemy compatible databases (currently supported flavors include MySQL, PostgreSQL, SQLite, and MS SQL) and is compatible with file backed store. See the `Tracking Store documentation <https://mlflow.org/docs/latest/tracking.html#storage>`_ for more information. (#756, @AndersonReyes; #800, #844, #847, #848, #860, #868, #975, @mparkhe; #980, @dbczumar)
```

Comment:
enables users to  -> enables you to

Reply:
Not sure it makes sense to call it a "community contribution feature" -- every contribution is from the community!

Reply:
Also, we could consider naming this "database store" instead. Users don't really care how it's implemented (whether it's SqlAlchemy or some other library) -- although I guess that might leak through in the connection strings and such that it supports.